README.md

```diff
@@ -1,650 +1,28 @@
-```
-
-
-1. __prop__ <code>{String}</code>:
-  The sortable property.
-1. __sortFunc__ <code>{function(sortPropValue, item1, item2)}</code>:
-  The
-  sortable behavior. The default behavior assumes the sort property value
-  looks like `PROPERTY DIRECTION` (ex: `name desc`).
-
-- __returns__ <code>{[Compares](#compares-objectstringproppropavalue-bvalue-a-b-prop-algebra)}</code>:
-  Returns a compares that can be used to create
-  a `set.Algebra`.
-
-
-### <code>set.props.id(prop)</code>
-
-
-Defines the property name on items that uniquely
-identifies them. This is the default sorted property if no
-[sort](#setpropssortprop-sortfunc) is provided.
-
-```js
-var algebra = new set.Algebra(set.props.id("_id"));
-algebra.index(
-  {sortBy: "name desc"},
-  [{name: "Meyer"}],
-  {name: "Adams"}) //-> 1
-
-algebra.index(
-  {sortBy: "name"},
-  [{name: "Meyer"}],
-  {name: "Adams"}) //-> 0
-```
-
-
-1. __prop__ <code>{String}</code>:
-  The property name that defines the unique property id.
-
-- __returns__ <code>{[Compares](#compares-objectstringproppropavalue-bvalue-a-b-prop-algebra)}</code>:
-  Returns a compares that can be used to create
-  a `set.Algebra`.
-
-
-## <code>new set.Translate(clauseType, propertyName)</code>
-
-
-Localizes a clause's properties within another nested property.
-
-```js
-var algebra = new set.Algebra(
-  new set.Translate("where","$where")
-);
-algebra.has(
-  {$where: {complete: true}},
-  {id: 5, complete: true}
-) //-> true
-```
-
-This is useful when filters (which are `where` clauses) are
-within a nested object.
-
-
-1. __clause__ <code>{String}</code>:
-  A clause type.  One of `'where'`, `'order'`, `'paginate'`, `'id'`.
-1. __propertyName__ <code>{String|Object}</code>:
-  The property name which contains the clauses's properties.
+## Documentation
 
-- __returns__ <code>{[Compares](#compares-objectstringproppropavalue-bvalue-a-b-prop-algebra)}</code>:
-  A set compares object that can do the translation.
+Read the [API docs on CanJS.com](https://canjs.com/doc/can-set.html).
 
+## Changelog
 
+See the [latest releases on GitHub](https://github.com/canjs/can-set/releases).
 
 ## Contributing
 
-To setup your dev environment:
+The [contribution guide](https://github.com/canjs/can-set/blob/master/CONTRIBUTING.md) has information on getting help, reporting bugs, developing locally, and more.
 
-1. Clone and fork this repo.  
-2. Run `npm install`.
-3. Open `test.html` in your browser.  Everything should pass.
-4. Run `npm test`.  Everything should pass.
-5. Run `npm run-script build`.  Everything should build ok.
+## License
 
-To publish:
+[MIT](https://github.com/canjs/can-set/blob/master/LICENSE.md)
 
-1.  Update the version number in package.json and commit and push this.
-2.  Run `npm publish`.  This should generate the dist folder.
-3.  Create and checkout a "release" branch.
-4.  Run `git add -f dist`.
-5.  Commit the addition and tag it `git tag v0.2.0`.  Push the tag `git push origin --tags`.
```

Comment:
This is a more concise version of that we already have in the `CONTRIBUTING.md` file, with two differences, it uses `test.html` in addition to `npm test`, and it includes publishing instructions. Should we update all `CONTRIBUTING.md`s to use this as well?